tests/native/django/settings.py

```diff
@@ -11,3 +11,19 @@
 INSTALLED_APPS = [
     'transifex.native.django',
 ]
+
+TEMPLATES = [
+    {
+        'BACKEND': 'django.template.backends.django.DjangoTemplates',
+        'DIRS': [],
+        'APP_DIRS': True,
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.request',
+                'django.contrib.auth.context_processors.auth',
+                'django.contrib.messages.context_processors.messages',
+            ],
+        },
+    },
+]
```

Comment:
What was this needed for?

Reply:
So that the new tests can work. Otherwise `Template(foo).compile(Context(bar))` raises an error.